### ISSUE TITLE
First assessment of new panan-01 and panan-005: Ready to save with higher frequency output?

### ISSUE THREAD

**Issue by schmidt-christina** [COSIMA/mom6-panan#48]
We now have 20 years of panan-01 and 7 years of panan-005, so I looked into some key metrics the assess them. If there are no major concerns about any of these, I would like to run panan-005 further and save daily output.

The shift of SWMT peak to lighter density classes in panan-01 is significantly reduced, but there is still a slight shift in the Adelie and Ross.

No shift of the peak SWMT density in panan-005, except for an initial shock, and higher SWMT and about 2 Sv higher cross contour transport in panan-005 compared to pnan-01.

![image](https://github.com/COSIMA/mom6-panan/assets/38216156/9ef6cfd5-c07f-4be2-a2f3-493b7e40a22c)

![image](https://github.com/COSIMA/mom6-panan/assets/38216156/ffccced5-be9a-4417-bcdb-5fd72925ee80)

Time series of othe rproperties can be found [here](https://github.com/schmidt-christina/mom6_comparison/blob/main/MOM6-005_assessment_timeseries.ipynb)
Some properties (e.g. MOC, T and S on the shelf) are still drifting but that's probably nothing too big. In the case of the T and S below 3000 m everywhere the drift is reduced a lot in panan-005 compared to panan-01. 

I think it looks ok for our purposes and I would continue panan-005 saving higher frequency output. I don't think I need to save anything with a higher frequency than daily as we are not using panan-005 to create movies?

- [ ]  Daily temperature: `thetao`
- [ ]  Daily salinity: `so`
- [ ]  Daily u,v: `uo,vo`
- [ ]  Daily mixed layer depth  `mlotst`
- [ ]  Monthly T advection zonal, meridional T_adx,T_ady
- [ ]  Monthly surface mass flux `wfo`
- [ ]  Monthly net surface boundary heat flux `hfds`
- [ ]  Monthly Net salt flux into ocean at surface (restoring + sea-ice) `salt_flux`
- [ ]  Daily mean SIC,SIT `siconc,sithick`
- [ ]  daily SSH `zos`
- [ ]  daily SSH snapshots `zos`

If there are no objections I will continue panan-005 at the end of the week and save all of the variables above in addition to what is already saved.

Something that is strange in both panan-01 and panan-005 is the age tracer at the northern boundary. It looks like it gets reset to 0 at the boundary. I haven't looked in the code at all yet whether we can change that. I wanted to use the age tracer at the bottom to trace the AABW pathway but now the signal of AABW gets overwhelmed by the inflow from the north.

https://github.com/COSIMA/mom6-panan/assets/38216156/8da68f70-df0f-4af2-9930-5a26bea821bd

**Comment by adele157** [comment]
Nice evaluation Christina.

Bugger about the age tracer, I guess we should have thought of that before! I think we may still be able to use it in the first ~10 years though, because it looks like there is still a separation between the young Antarctic-sourced waters and the low age coming from the north. Much beyond 10 years and that might not be the case though.

I think it would be worthwhile to have a meeting with @schmidt-christina, @willaguiar and I (and anyone else who would like to join) to think about exactly what analysis we envisage doing with the output from these runs and make sure we have everything covered. e.g. some additional diags we might want to consider are:
* daily depth integrated x,y heat transport (2D)
* daily heat, freshwater and salt fluxes - would these be better for more accurate SWMT? (2D)
* monthly sea ice transport
* Do we have any mixing diagnostics turned on? If not, maybe we want monthly 3D net diffusivity to look at mixing along AABW pathways? Or are there other diagnostics that would be useful to get at this question?

What do we envisage using daily u, v, temp, salt for? Would we be better off with daily mass transports on density layers instead of daily u and v? That would enable us to look at high frequency variability in bottom water, and probably we could still use them for ASC studies?

**Comment by schmidt-christina** [comment]
Good idea to meet Adele, but I will only be working until next Thursday, before taking two days off and then flying to Germany.

I haven't looked into which mixing diagnostics to turn on, but that's on my list of things to check.

@MatthisAuger should probably join such a meeting as he wants to use panan-005 and needs daily u,v,T,S

**Comment by MatthisAuger** [comment]
Would be interested to join such a meeting indeed!

I probably could do part of the analysis I'm planning to do with daily mass transport on density layer. Daily T,S at least would still be very useful for studying the dense water overflows, so I can evaluate the contribution of steric height on the SSH signal and look at the T/S variability of the overflows.

**Comment by willaguiar** [comment]
Agreed on the meeting

For my Antarctic Slope Current project, Ill need mostly daily depth integrated x,y heat transport (2D), and perhaps daily u,v , or depth integrated mass transports

**Comment by schmidt-christina** [comment]
A bit more than 1 year of high frequency output is available now. You can access it using this database `/g/data/e14/cs6673/mom6_comparison/Python_scripts/panan_005deg_jra55_ryf.db` using the expt name `panan_005deg_jra55_ryf`. The data is here `/scratch/e14/cs6673/mom6/archive/panan_005deg_jra55_ryf_2023_05_17/` with daily output starting in `output108`. Only the output unitil `output120` is cleaned up atm, so the first year. I will clean up the output of year 2 and 3 when it's available next week.

**Comment by adele157** [comment]
I also transferred the latest panan-01 data here (overwriting the old data
there):

/g/data/ik11/outputs/mom6-panan/panant-01-zstar-ACCESSyr2/


I have yet to rerun the years with high frequency output.

**Comment by MatthisAuger** [comment]
Hi @schmidt-christina and thanks again for the outputs.

I get an error when trying to open them from my Jupyter Notebook sessions. 
If I try `cc.database.create_session('/g/data/e14/cs6673/mom6_comparison/Python_scripts/panan_005deg_jra55_ryf.db')`
I get `OperationalError: (sqlite3.OperationalError) unable to open database file`

I can't access it either when trying to directly open the ncfiles from your folders.

It might be a file permission issue?

**Comment by adele157** [comment]
@MatthisAuger Christina is away on leave at the moment. I would suggest requesting to join the e14 project (and send Rishav Goyal and Matt England an email explaining you need to access the data). The files have e14 group reading permissions on scratch.

@schmidt-christina when you're back it would be good to transfer the output to ik11.

**Comment by MatthisAuger** [comment]
Oh sorry my mistake actually. Switched to ARE recently and forgot I had to add e14 manually when creating a session.

**Comment by MatthisAuger** [comment]
I think my problem come from the data being on /scratch/e14 and not on /g/data/e14. I can access /scratch/e14 from my terminal but not from my jupyter notebook. On the later one I can only access /g/data/e14. Does that make sense?

**Comment by adele157** [comment]
Did you add access to scratch/e14 in the storage line as well as gdata?

**Comment by MatthisAuger** [comment]
Much better indeed. Sorry about that and thanks @adele157

**Comment by schmidt-christina** [comment]
All data is now on ik11 in

`/g/data/ik11/outputs/mom6-panan/panant-005-zstar-ACCESSyr2`
